endpoints/orders.yaml

```diff
@@ -290,229 +290,71 @@ paths:
-            rejectReviseOrderEvent:
-              '{$request.body#/rejectReviseOrderResponseUrl}':
-                post:
-                  summary: Event from SGS that the revise order request was rejected
-                  description: Event from SGS that the revise order request was rejected
-                  requestBody:
-                    description: rejectReviseOrderResponseUrl callback
-                    required: true
-                    content:
-                      application/json:
-                        schema:
-                          $ref: 'schemas/commonSchemas.yaml#/components/schemas/encodededPayloadWrapper'
-                  responses:
-                    '202':
-                      description: Request processed successfully
-                      content:
-                        application/json:
-                          schema:
-                            $ref: 'schemas/commonSchemas.yaml#/components/schemas/acceptedResponse'
-  /v1/orders/unsubscribe:
-        delete:
-          summary: Unsubscribe to order events
-          description: Unsubscribe to order events
-          operationId: ordersUnsubscribe
-          requestBody:
-            description: ordersUnsubscribe
-            required: true
+
+  /v1/orders/subscriptions/{id}:
+      delete:
+        summary: Remove a subscription
+        description: Existing subscriptions can be deleted by making a DELETE request against the subscription resource
+        operationId: Remove a subscription
+        parameters:
+        - name: id
+          in: path
+          description: The subscription id
+          required: true
+        responses:
+          '204':
+            description: Subscription deleted
+          default:
+            description: Unexpected error occured
             content:
               application/json:
                 schema:
-                  oneOf:
-                    - $ref: 'schemas/ordersSchemas.yaml#/components/schemas/ordersSubscribeSGS'
-                    - $ref: 'schemas/ordersSchemas.yaml#/components/schemas/ordersSubscribeSGSCounterParty'
-          responses:
-            '202':
-              description: Request processed successfully
-              content:
-                application/json:
-                  schema:
-                    $ref: 'schemas/commonSchemas.yaml#/components/schemas/acceptedResponse'
-            default:
-              description: Unexpected error occured
-              content:
-                application/json:
-                  schema:
-                    $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'
\ No newline at end of file
+                  $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'
+      
+      get:
+        summary: Enumerating subscriptions
+        description: Get a list of active subscriptions
+        operationId: Enumerating subscriptions
+        parameters:
+        - name: id
```

Comment:
not sure supplying subscription id makes sense. thought the immediate use case of enumerate subscriptions was to get the ids to facilitate a delete.